tensorboard/plugins/debugger_v2/debug_data_multiplexer.py

```diff
@@ -36,9 +37,41 @@
 # ones are either repetitions of the earlier ones or caused by the earlier ones.
 DEFAULT_PER_TYPE_ALERT_LIMIT = 1000
 
+# Default interval between successive calls to `DebugDataReader.update()``.
+DEFAULT_RELOAD_INTERVAL_SEC = 60
 
-def run_in_background(target):
-    """Run a target task in the background.
+
+class Timer(object):
+    """A timer that supports interruptible sleep()."""
+
+    def __init__(self, interval_sec):
```

Comment:
It seems kind of arbitrary to take the interval as a constructor parameter when it would be simpler and more flexible to just pass it into `sleep()` - this would matter I suppose if we had many different calls to `sleep()` but this is in fact called in just a single place anyway, so we could reduce the length through which this constant gets plumbed by just passing it directly there.

Reply:
The revised `run_repeatedly_in_background()` now takes an argument for the sleeping interval (`interval_sec`).